scripts/helm-template-snapshot.sh

```diff
@@ -54,6 +54,7 @@ update() {
       # shellcheck disable=SC1090
       source "$CONFIG_FILE"
       generate "$RELEASE_NAME" "$CHART" "$VALUES_FILE" >"$TARGET_FILE"
+      kubeval "${TARGET_FILE}"
```

Comment:
I suspect we'd want to use the `--strict` flag here, [docs](https://www.kubeval.com/#strict-schemas`

```suggestion
      kubeval --strict "${TARGET_FILE}"
```